operations/app/src/modules/front_door/main.tf

```diff
@@ -85,30 +94,24 @@ resource "azurerm_frontdoor" "front_door" {
     dynamic "frontend_endpoint" {
         for_each = var.https_cert_name != null ? [1] : []
         content {
-            name = var.https_cert_name
-            host_name = replace(var.https_cert_name, "-", ".") // This will change test-prime-cdc-gov to test.prime.cdc.gov
-            custom_https_provisioning_enabled = true
-            custom_https_configuration {
-                certificate_source = "AzureKeyVault"
-                azure_key_vault_certificate_secret_name = var.https_cert_name
-                azure_key_vault_certificate_secret_version = data.azurerm_key_vault_secret.https_cert[0].version
-                azure_key_vault_certificate_vault_id = var.key_vault_id
-            }
+          name = var.https_cert_name
+          host_name = replace(var.https_cert_name, "-", ".") // This will change test-prime-cdc-gov to test.prime.cdc.gov
+          custom_https_provisioning_enabled = true
+          
+          custom_https_configuration {
+            certificate_source = "AzureKeyVault"
+            azure_key_vault_certificate_secret_name = var.https_cert_name
+            azure_key_vault_certificate_secret_version = data.azurerm_key_vault_secret.https_cert[0].version
+            azure_key_vault_certificate_vault_id = var.key_vault_id
+          }
         }
-    }
+      }
```

Comment:
I think we have inconsistent IDE settings for Terraform files, as this looks like an auto-formatting change. I'm going to propose an editorconfig for *.tf files in an upcoming PR so we don't end up going back and forth adjusting formatting between our environments